apps/src/code-studio/pd/form_components/button_list.jsx

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import _ from 'lodash';
-import {Radio, Checkbox, ControlLabel, FormGroup, HelpBlock} from 'react-bootstrap';
+import {
```

Comment:
Is this a style preference anywhere? If so, should we be enforcing this somewhere?